### PR TITLE
Nonroot scratch

### PR DESCRIPTION
**Pull request by Ealdwulf**
This PR adds a systemd config file allowing user processes launched by `systemd --user` to obtain real-time priority.
This allows scratch to run as the ordinary user: For https://github.com/KanoComputing/peldins/issues/2385

Another PR will be needed to clean up files made in the wrong place/owned by root due to running scratch as root.
@skarbat @alex5imon

### CODE REVIEW

**Comment by alex5imon** [comment]
nice!

File: config/systemd/kano.conf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Copyright (C) 2016 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPLv2
+
+# Enable processes run by sysemd on behalf of the user to use
```
Comment (alex5imon) [inline config/systemd/kano.conf]: typo?

**Comment by skarbat** [comment]
Nice one @Ealdwulf !

File: config/systemd/kano.conf (hunk 1 above)
Comment (Ealdwulf) [inline config/systemd/kano.conf]: Thanks, fixed :-)

**Pull request merged**